demo/full/scripts/components/GeneratedLink.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export default function GeneratedLink({
```

Comment:
Also this name does not correspond to what the component is.

It is not the generated link but a button to display the generated link.

Ex: `GenerateLinkButton`